portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -18,6 +18,7 @@
 import java.io.IOException;
 import java.util.Arrays;
```

Comment:
you imported Arrays twice here? Also I'm not sure if I'm missing it but I don't see you use Arrays in the code. If so, just delete the imports